### DISCUSSION TITLE
Using Chrome's "New tab to the right" causes some tabs to the right to become root nodes

### DISCUSSION THREAD

**Discussion by airliftbluegill** [Able-Bear-Software/tab-nodes-tree#70] [Bug Report]
Example:
With the highlighted tab opened:

![image](https://github.com/user-attachments/assets/4ddfb88e-31e3-4fa9-bbf4-52be12803907)

Right click the tab at the top of the window, and choose "New tab to the right":

![image](https://github.com/user-attachments/assets/b70e7084-0951-421d-8d5c-847cdb5d7b89)

Some tabs to the right are now root nodes, rather than staying where they were:

![image](https://github.com/user-attachments/assets/02f845ff-d82a-48b6-a436-a1e85555566f)

**Comment by zeroarst** [comment]
@airliftbluegill Thank you for reporting this issue.
I have created a ticket: https://github.com/Able-Bear-Software/tab-nodes-tree/issues/72.
However, this will be a low priority, as a new tab node can already be created using TNT shortcuts (chrome://extensions/shortcuts), the built-in shortcut (Ctrl+T), or the right-click context menu. The goal of this extension is to replace native tab bars, though feature parity has not yet been fully achieved.
